.github/workflows/validate-licenses.yaml

```diff
@@ -39,3 +39,37 @@ jobs:
           args: validate container-images-mapping --input=images.txt --mapping-file=licenses.d2iq.yaml --check-sources --output-format=github
```

Comment:
we removed the `continue-on-error: true` from the `runValidation` step above, so if that fails, the images won't be updated.

I'm not sure if we want to add it back while keeping this as opt-in with the label because we want the job to fail if the images aren't updated. If we added that back, the validation would fail but it would continue to the next step which is optional based on the label and the job would be considered successful. So PRs that do not have that label would never fail the license check step 🤔

Reply:
an alternative might be to create a separate job that does the update if the label is added and then make sure that the validation step is always run